docs/source/release/v3.9.0/sans.rst

```diff
@@ -10,7 +10,7 @@ Features
 
 - Make added multi-period event files available for the ISIS SANS reduction.
 - Added NXcanSAS as an output format of the ISIS SANS Gui.
-
+- Added MASK/LINE for LOQ
```

Comment:
The dot at the end is missing. These should be complete sentences, think. Perhaps add a few words what this MASK/LINE does, as well?